File: templates/csharp/team_source/Strategy.cs
public static class Strategy
{
    public static string ChooseMove(
        int turn,
        string myHistory,
        string opponentHistory,
        RpsRandom rng)
    {
        string[] moves = { "R", "P", "S" };
        return moves[rng.NextInt(moves.Length)];
    }
}

File: templates/rust/team_source/strategy.rs
pub fn choose_move(
    _turn: usize,
    _my_history: &str,
    _opponent_history: &str,
    rng: &mut RpsRandom
) -> &'static str {
    let moves = ["R", "P", "S"];
    moves[rng.next_usize(moves.len())]
}
